app/uk/gov/hmrc/tradergoodsprofilesrouter/models/request/CreateRecordRequest.scala

```diff
@@ -53,7 +53,9 @@ object CreateRecordRequest {
       (JsPath \ "countryOfOrigin").read(lengthBetween(1, 2).andKeep(verifying(isValidCountryCode))) and
       (JsPath \ "category").read[Int](verifying[Int](category => category >= 1 && category <= 3)) and
       (JsPath \ "assessments").readNullable[Seq[Assessment]] and
-      (JsPath \ "supplementaryUnit").readNullable[Int] and
+      (JsPath \ "supplementaryUnit").readNullable[Int](
+        verifying[Int](supplementaryUnit => supplementaryUnit >= -2147483648 && supplementaryUnit <= 2147483647)
```

Comment:
Should be able to replace those constants with Int.MaxValue and Int.MinValue

Feels like there must be a nicer way to do this that isn't manually checking the bounds though... a conversion perhaps